Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -213,6 +192,97 @@ public struct Configuration: Equatable {
     }
 }
 
+private func validateRuleIdentifiers(configuredRules: [Rule], disabledRules: [String]) -> [String] {
+    // Validate that all rule identifiers map to a defined rule
+    let validRuleIdentifiers = configuredRules.map { type(of: $0).description.identifier }
+
+    let invalidRules = disabledRules.filter { !validRuleIdentifiers.contains($0) }
+    if !invalidRules.isEmpty {
+        for invalidRule in invalidRules {
+            queuedPrintError("configuration error: '\(invalidRule)' is not a valid rule identifier")
+        }
+        let listOfValidRuleIdentifiers = validRuleIdentifiers.joined(separator: "\n")
+        queuedPrintError("Valid rule identifiers:\n\(listOfValidRuleIdentifiers)")
+    }
+
+    return validRuleIdentifiers
+}
+
+private func containsDuplicatedRuleIdentifiers(_ validDisabledRules: [String]) -> Bool {
+    // Validate that rule identifiers aren't listed multiple times
+    if Set(validDisabledRules).count != validDisabledRules.count {
+        let duplicateRules = validDisabledRules.reduce([String: Int]()) { accu, element in
+            var accu = accu
+            accu[element] = (accu[element] ?? 0) + 1
+            return accu
+        }.filter { $0.1 > 1 }
+        queuedPrintError(duplicateRules.map { rule in
+            "configuration error: '\(rule.0)' is listed \(rule.1) times"
+        }.joined(separator: "\n"))
+        return true
+    }
+
+    return false
+}
+
+private func handleAlias(_ alias: String, ruleList: RuleList) -> String {
```

Comment:
since this is only used in the `handleAliasWithRuleList` closure above, could you fold it into the closure and remove the function?